e2e_testing/torchscript/backprop.py

```diff
@@ -43,10 +45,33 @@ def __init__(self):
         ([-1, -1], torch.float32, True),
         ([-1, -1], torch.float32, True),
     ])
+    def forward(self, grad_out, output):
+        return torch.ops.aten.tanh_backward(grad_out, output)
 
-    def forward(self, out_grad, output):
-        return torch.ops.aten.tanh_backward(out_grad, output)
 
 @register_test_case(module_factory=lambda: TanhBackwardModule())
 def TanhBackward_basic(module, tu: TestUtils):
     module.forward(torch.randn(3, 3), torch.randn(3, 3))
+
+
+# ==============================================================================
+# This test can be removed once we have one real op returning 3 float32 tensors
+class ReturnThreeMemrefF32(torch.nn.Module):
```

Comment:
Don't mention memref. Also maybe put this in basic.py since it isn't related to backprop